marklogic-data-hub/src/main/java/com/marklogic/hub/hubcentral/conversion/FlowConverter.java

```diff
@@ -514,4 +524,23 @@ protected void transformMatchingOptions(ObjectNode stepNode) {
             stepNode.set(fieldName, node);
         }
     }
+
+    protected void transformMergingOptions(ObjectNode stepNode) {
+        JsonNode beforeNode = stepNode.get("mergeOptions");
+        // if mergeOptions is null, this conversion has already run
+        if (beforeNode == null) {
+            return;
+        }
+
+        JsonNode afterNode = MasteringService.on(hubConfig.newHubClient().getStagingClient()).updateMergeOptions(beforeNode);
```

Comment:
No need to change this, but just an FYI, pinging @srinathgit  and @ryanjdew  on this too - one thing I'd like to do, particularly in our tests, is get a HubClient as early as possible and use that for all connections. The benefit is that it doesn't create a new DatabaseClient each time the way HubConfig does when you call one of its new*Client methods. That also means less and less code that depends on HubConfig - I'd really like our code to depend on HubClient and (if needed) HubProject, with HubConfig becoming something that's discarded once those two things are obtained. But again, no need to change this, just a note.